### PR TITLE
Piechart

### PR DESCRIPTION
**Pull request by ferni**
# [Create Piechart component](https://app.shortcut.com/oazo-apps/story/3790/create-piechart-component)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Add PieChart component.
- Add PieChart story.
- Remove faulty implementation of i18n support in Storybook.
  
## How to test 🧪
  <Please explain how to test your changes>

- After checking the code you can check the built storybook here: https://piechart-storybook.surge.sh/
- It looks like this:
Screen Shot 2022-04-27 at 23 02 31

### CODE REVIEW

File: blockchain/tokensMetadata.ts [hunk 1]
```diff
@@ -15,8 +15,12 @@ export interface TokenConfig {
   tags: CoinTag[]
   color: string
   bannerIcon: string
+  bannerGif?: string
```
Comment (BackToTheCode) [inline blockchain/tokensMetadata.ts:18]: Are these additional props connected with this PR?

File: components/dumb/PieChart.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      angle,
+      color: items[i].color,
+    })
+    angle += 360 * ratio
+  }
+  return slices
+}
+
+export function PieChart({ items, size = 258 }: { items: PieChartItem[]; size?: number }) {
+  const strokeWidth = 34
```
Comment (BackToTheCode) [inline components/dumb/PieChart.tsx:33]: Could possibly have a variable strokeWidth that scales with chart size?

**Review by BackToTheCode — APPROVED** [review]
Great stuff. And you fixed Storybook. Have left two thoughts

File: components/dumb/PieChart.tsx (hunk 2 above)
Comment (BackToTheCode) [reply components/dumb/PieChart.tsx:33]: Build is failing on vercel. But I can't access logs

Comment (ferni) [reply components/dumb/PieChart.tsx:33]: Ah nice catch, yes it should probably scale, I'll see if I can do that.

File: blockchain/tokensMetadata.ts (hunk 1 above)
Comment (ferni) [reply blockchain/tokensMetadata.ts:18]: They're not actually, just some properties that I think were missing. I attempted something using this file but then reverted, so they're kind of left overs.

**Review by sennett — APPROVED** [review]

**Pull request merged**